test/libsolidity/syntaxTests/fallback/fallback_as_function_name.sol

```diff
@@ -0,0 +1,5 @@
+contract C {
+    function fallback() external pure {}
+}
+// ----
+// ParserError: (26-34): This function is named "fallback" but is not the fallback function of the contract. If you intend this to be a fallback function, use "fallback(...) { ... }" without the "function" keyword to define it.
```

Comment:
Do we really want these two to be errors?

Reply:
I'll make them warnings.